### ISSUE TITLE
Compilation problem

### ISSUE THREAD

**Issue by chejunwei2** [ttadano/alamode#206]
Dear developer,
I compiled the latest alamode, but I got an error: 
```
make: Warning: File `writer.cpp' has modification time 35371 s in the future
icpc  -O2 -xHOST -qopenmp -std=c++11 -I../include -I/home/chejunwei/software/library/spglib-1.9.7/include/spglib -c writer.cpp
writer.cpp(125): error: class "ALM_NS::OptimizerControl" has no member "mirror_image_conv"
          std::cout << "  MIRROR_IMAGE_CONV = " << optctrl.mirror_image_conv << "\n\n";
                                                           ^
compilation aborted for writer.cpp (code 2)
make: *** [writer.o] Error 2
```
So I would like to ask for your help. Thank you very much.
Best regards,
Junwei Che

**Comment by ttadano** [comment]
Hi,

I did not reproduce this error with the latest release (v.1.5.0) and the develop branch version.

Can you tell me the exact version number you are using?

**Comment by chejunwei2** [comment]
Dear developer,
I'm trying to install the merged version of 1.5.0 and shengbte_4ph.  Furthermore, I used icpc-18.0.5, boost-1.7.8, eigen-3.4.0, and spglib-1.9.7. 
Thank you so much for your help.
Best regards

**Comment by ttadano** [comment]
I have merged `shengbte_4ph` branch into `develop`. Please use the latest `develop` branch for the moment.
